### PR TITLE
2.17 documentation

### PR DESCRIPTION
**Pull request by kflynn**
Signed-off-by: Flynn <flynn@buoyant.io>

### CODE REVIEW

File: linkerd.io/content/2.17/features/egress.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+title: Egress
+---
```
Comment (travisbeckham) [inline linkerd.io/content/2.17/features/egress.md:3]: Can we add a short description here? It's need for the Features listing page.

Same for the edge doc.

File: linkerd.io/content/2.17/reference/egress-network.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
+---
+title: EgressNetwork
+---
```
Comment (travisbeckham) [inline linkerd.io/content/2.17/reference/egress-network.md:3]: Can we add a short description here? It's needed for the list page.

Same for the edge doc.

**Review by travisbeckham — APPROVED** [review]
Looks great!
I added a couple comments on the new feature docs, then I think it's ready to go.

File: linkerd.io/content/2.17/features/egress.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+by itself; for example, a malicious actor could bypass the Linkerd sidecar -
+and thus Linkerd's egress controls - entirely. Fully restricting egress
+traffic in the presence of arbitrary applications thus typically requires a
+more comprehensive approach.
+
+{{< /warning >}}
+
+Related content:
+
+* [Guide:  Managing egress traffic]
```
[the remark this answers is not in the document]
Comment (kflynn) [reply linkerd.io/content/2.17/features/egress.md:24]: Nice catch, thanks!

File: linkerd.io/content/2.17/tasks/managing-egress-traffic.md [hunk 4]
```diff
@@ -0,0 +1,443 @@
...
+
+```bash
+kubectl apply -f - <<EOF
+apiVersion: gateway.networking.k8s.io/v1alpha2
+kind: HTTPRoute
+metadata:
+  name: httpbin-get
+  namespace: egress-test
+spec:
+  parentRefs:
```
Comment (bob-bins) [inline linkerd.io/content/2.17/tasks/managing-egress-traffic.md:152]: aren't you forgetting `hostnames` here? (similarly for the httproute near the bottom of this page)
```
  hostnames:
    - httpbin.org
```

Comment (bob-bins) [reply linkerd.io/content/2.17/tasks/managing-egress-traffic.md:152]: @kflynn in case you didn't get a notification for the comment above

**Pull request merged**